README.md

```diff
@@ -16,16 +16,19 @@ for details.
 
 * Thrustmaster T500RS
 * Thrustmaster TX
-
+* Thrustmaster TMX
 See the status of additional devices on
 [GitHub Projects](https://github.com/her001/tmdrv/projects/1).
 
 ## Requirements
 
 * [Python 3](https://www.python.org)
 * [python-libusb1](https://pypi.python.org/pypi/libusb1)
-* jscal (from [Linux Console](https://sourceforge.net/projects/linuxconsole),
-for calibration)
+* jscal 
+ - Ubuntu: sudo apt install joystick
+ - Other Distro's: compile from source from [Linux Console](https://sourceforge.net/projects/linuxconsole),
+for calibration) 
+
```

Comment:
I would like to propose an alternative. Below the "Requirements" section of the readme could be an "Installation" section with instructions on how to install all dependencies for various distro's and set the udev rules. This will make the utility much more accessible as not everything is straightforward. Other people can then contribute to the Installation instructions for other distro's. 
For example.


Installation

Ubuntu: 
Install the dependencies:
sudo apt install python-libusb1 joystick git

Create the udev rules:
sudo echo "SUBSYSTEM=="\""usb"\"", ATTRS{idVendor}=="\""044f"\"", MODE:="\""0666"\"", GROUP:="\""users"\""" >> /etc/udev/rules.d/10-local.rules
sudo udevadm control --reload
sudo udevadm trigger --action=add

Clone the repository:
git clone https://github.com/her001/tmdrv.git